Doc/library/math.rst

```diff
@@ -646,8 +646,27 @@ Constants
 
 .. data:: nan
 
-   A floating-point "not a number" (NaN) value.  Equivalent to the output of
-   ``float('nan')``.
+   A floating-point "not a number" (NaN) value. Equivalent to the output of
+   ``float('nan')``. Specifically, due to the requirements of the `IEEE-754 standard
+   <https://en.wikipedia.org/wiki/IEEE_754>`_, ``math.nan`` and ``float('nan')`` are
+   not considered to equal to any other numeric value, including themselves. To avoid errors
+   when checking whether a number is a NaN, use the :func:`isnan` function to test
```

Comment:
```suggestion
   not considered to equal to any other numeric value, including themselves. To check
   whether a number is a NaN, use the :func:`isnan` function to test
```

"errors" is confusing because it suggests an exception would be thrown otherwise.